website/docs/styleguide.md

```diff
@@ -25,6 +25,610 @@ sidebar:
         url: https://github.com/explosion/spaCy/blob/master/CONTRIBUTING.md
 ---
 
-import Readme from 'README.md'
```

Comment:
The `menu` on top of this page needs to be adjusted to remove the `Setup & Installation` section

Reply:
Solved by [2ffecb8](https://github.com/explosion/spaCy/pull/11855/commits/2ffecb8b9f4cdef79233030dc8b77f41b0c41323)

Reply:
(FYI feel free to resolve yourself, Marcus!)

Reply:
Will do next time.